fastly/__init__.py

```diff
@@ -0,0 +1,6 @@
+registry = []
+
+
+def register_function(func):
```

Comment:
I'm open to suggestions here. I think `register_function()` is too generic.

Reply:
naming things... hmmmm. I don't have any improvement.